lib/src/create_report.dart

```diff
@@ -553,11 +559,12 @@ Future<ReportSection> _trustworthyDependency(
               ],
           ['**Transitive dependencies**'],
           for (final package in outdated.packages)
-            if (!pubspec.dependencies.containsKey(package.package))
+            if (!pubspec.dependencies.containsKey(package.package) &&
```

Comment:
Perhaps add a comment with a link to:
https://github.com/dart-lang/pub/issues/2552